library/Readme.md

```diff
@@ -2,6 +2,6 @@
 
 ### ffmpeg
 
-Current version: 0.44.6
+Current version: 0.47.0
 
-Updating: Download prebuilt libraries from https://github.com/iteufel/nwjs-ffmpeg-prebuilt/releases.
+Updating: Download prebuilt libraries from https://github.com/iteufel/nwjs-ffmpeg-prebuilt/releases
```

Comment:
Why are you removing the punctuation from this sentence? ;-)

Reply:
@mikeller I think that this punctuation is breaking the link when you tried to open it and is better to remove it, what do you think?

Reply:
Weird - this is working fine for me here - what operating system / browser do you have the problem with?

Reply:
ups sorry, tested it now in google chrome and works, maybe i have tested it directly from Notepad++ not sure, fixed!